airbyte-integrations/connectors/source-intercom/unit_tests/test_cached_stream_state.py

```diff
@@ -0,0 +1,81 @@
+#
```

Comment:
this file was reset by reverting [this PR](https://github.com/airbytehq/airbyte/pull/23013)